samples/plugins/postgres_jdbc/connectionFields.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<connection-fields>
+  <field name="server" label="Server" value-type="string" category="endpoint" >
+    <validation-rule reg-exp="^(([a-zA-Z]|[a-zA-Z][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z]|[A-Za-z][A-Za-z0-9\-]*[A-Za-z0-9])$"/>
+  </field>
+
+  <field name="port" label="Port" value-type="string" category="endpoint" default-value="5432" />
+
```

Comment:
You need a non-editable `authentication` field that has a default value of `auth-user-pass`

Reply:
I wonder if we should add this as a check in the packager. Should be pretty easy to do since we keep track of the connection fields now.

Reply:
I thought we did, but apparently not. Yes that's a good idea.